### PR TITLE
mep12 on multiple_figs_demo.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/multiple_figs_demo.py [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+ax.set_xticklabels([])
 
-figure(1)
-savefig('fig1')
-figure(2)
-savefig('fig2')
+plt.figure(1)
+plt.savefig('fig1')
+plt.figure(2)
+plt.savefig('fig2')
```
Comment (WeatherGod) [inline examples/pylab_examples/multiple_figs_demo.py]: I wonder if these savefigs are worth keeping around. I think the lesson of the example is clear enough without them.

Comment (thisch) [inline examples/pylab_examples/multiple_figs_demo.py]: I agree with you.

Comment (ericmjl) [inline examples/pylab_examples/multiple_figs_demo.py]: Okay, I will take them out then.

**Comment by ericmjl** [comment]
What's the error here? I don't see pep8 issues, and local test shows the code works just fine.

**Comment by dopplershift** [comment]
It looks to me like a transient failure. I restarted that particular test, so we can see if that passes now.

**Comment by dopplershift** [comment]
Looks like that fixed it.

**Pull request merged**